tests/dbt/test_adapter.py

```diff
@@ -148,9 +148,10 @@ def test_adapter_dispatch(sushi_test_project: Project, runtime_renderer: t.Calla
     context = sushi_test_project.context
     renderer = runtime_renderer(context)
     assert renderer("{{ adapter.dispatch('current_engine', 'customers')() }}") == "duckdb"
+    assert renderer("{{ adapter.dispatch('current_engine')() }}") == "duckdb"
 
-    with pytest.raises(ConfigError, match=r"Macro 'current_engine'.*was not found."):
```

Comment:
This doesn't look right to me. Why did this test change? The dispatch call should've failed since  the `current_engine` macro is in the `customers` package.

Reply:
`current_engine` is a dispatch, so it resolves if in another package. We can narrow what the dispatch finds if we don't want this. For example, maybe if the requested package is `dbt.*`, we look in all `dbt.*` packages. Otherwise, we only look in the requested package?

Reply:
I think we absolutely need to take the package into account. I wouldn't hardcode the `dbt` package though.